tuning/autotune.py

```diff
@@ -511,7 +526,7 @@ def benchmark_top_candidates(
     )
 
     logging.critical(
-        f"Total: {len(benchmark_results)} | Benchmarked: {len(benchmarked_files)} | Failed: {len(benchmark_failed_files)} | Benchmarking Rate: {len(benchmarked_files)/(len(benchmark_results)*100):.1f}%"
+        f"Total: {len(benchmark_results)} | Benchmarked: {len(benchmarked_files)} | Failed: {len(benchmark_failed_files)} | Benchmarking Rate: {(len(benchmarked_files)/len(benchmark_results))*100}%"
```

Comment:
why should we change this?

Reply:
old division is `a / (b*100)`, should change to `(a/b) * 100`